utils/apollo-client.js

```diff
@@ -0,0 +1,18 @@
+const { createHttpLink } = require('apollo-link-http');
+const { InMemoryCache } = require('apollo-cache-inmemory');
+const { ApolloClient } = require('apollo-boost');
+require('dotenv').config();
+
+const fetch = require('node-fetch');
+
+const client = new ApolloClient({
+  link: createHttpLink({
+    uri: process.env.BASE_URI,
```

Comment:
Add prefix `TEST`, like `TEST_BASE_URI` to show that it is an environment variable related to the testing

Reply:
fixed